core/src/main/java/ru/urururu/sanity/cpp/tools/Language.java

```diff
@@ -10,6 +10,7 @@
     Cpp("cpp"),
     ObjectiveC("m"),
     Swift("swift"),
+    Rust("rs"),
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00115)